src/config.py

```diff
@@ -7,4 +7,10 @@
 TOKEN = os.environ.get("TOKEN")  # required
 DEBUG = os.environ.get("DEBUG", False)
 
+ROLE_IDS = [
+  ("Webgroup", "1166751688598761583"),
+  ("Gamez", "1089204642241581139"),
+  ("Croomer", "1172696659097047050"),
+  ]

```

Comment:
Do we think this should be a dictionary, so individual roles can be accessed? You can still use a list comprehension over the dict key:values, in the plugin file.

Reply:
as i see it userroles.py accesses roles specially formatted as hikari.CommandChoices so the array needs to specifically contain such a struct. i have to convert it at some point; nevertheless the original pairs cam be accessed from config.py

even in the error handlers below it infers role from context.